python/cog/server/http.py

```diff
@@ -104,6 +104,52 @@ def healthcheck() -> Any:
                 "setup": app.state.setup_result_payload,
             }
         )
+    
+    @app.get("/health/ready")
+    def healthcheck_readiness() -> Any:
+        _check_setup_result()
+        health = app.state.health
+        
+        if health == Health.UNKNOWN:
+            return JSONResponse(
+                {"detail": "Unknown server status"}, status_code=418
```

Comment:
Instead of using teapot status code, should we use 500?

Reply:
No XD (but yeah I'll change it)